content/influxdb/v2.0/reference/rc0-upgrade-guide.md

```diff
@@ -0,0 +1,271 @@
+
+## 6. Configure configuration profiles for the InfluxDB CLI
+
+Next, set up your InfluxDB CLI to connect to your old and new instances.
+If you've used the CLI before, you can copy your existing config file to your new data directory:
+
+```sh
+cp ~/.influxdbv2_old/configs ~/.influxdbv2/configs
+```
+
+You might want to edit that file and rename your old config to be something to indicate that it's the old version, like `influx_old`.
+
+{{< keep-url >}}
+```toml
+[influx_old]
+  url = "http://localhost:9999"
+  token = "<YOUR TOKEN>"
+  org = "influxdata"
+  active = true
+```
+
+If you've never used the CLI before, you can create a new configuration profile using the `influx config` command.
+
+```sh
+influx config create \
+    --config-name influx_old \
+    --host-url http://localhost:9999 \
+    --org influxdata --token <OLD_TOKEN>
+```
+
+If you run `influx config ls` after this, you should see a profile for your old instance.
+
+```sh
+$ influx config ls
+Active  Name        URL                    Org
+*       influx_old  http://localhost:8086  InfluxData
+```
+
+Next, let's set up your new instance which will automatically create a configuration profile for you.
+You can use the same username and password as your old instance, or something completely new, it's up to you.
+
+{{% warn %}}
+For a bucket name, make sure you don't use the same name as a bucket in your existing instance, otherwise, there will be a collision when you try to copy your resources over.
+We will delete this dummy bucket after everything is moved over.
+{{% /warn %}}
+
+```
+$ influx setup
+Welcome to InfluxDB 2.0!
+Please type your primary username: admin
+
+Please type your password:
+
+Please type your password again:
+
+Please type your primary organization name: InfluxData
+
+Please type your primary bucket name: dummy_bucket
+
+Please type your retention period in hours.
+Or press ENTER for infinite.:
+
+You have entered:
+  Username:          admin
+  Organization:      InfluxData
+  Bucket:            dummy_bucket
+  Retention Period:  infinite
+Confirm? (y/n): y
+
+Config default has been stored in /Users/rsavage/.influxdbv2/configs.
+User	Organization	Bucket
+admin	InfluxData	dummy_bucket
+```
+
+You now have two config profiles: one named `default` which points to your new instance, and one named `influx_old` that points to your old instance.
+
+```sh
+$ influx config ls
+Active	Name		URL					Org
+ 	default		http://localhost:8086			InfluxData
+*	influx_old	http://localhost:9999			InfluxData
+```
+
+Now you can send commands to each of them as needed using the `-c, --active-config` option on the CLI.
+
+## 7. Copy all resources from old instance to the new one
+
+Now we can copy all your existing influxdb resources, such as dashboards, tasks, and alerts to your new instance using a single command.
+
+```sh
+influx export all -c influx_old | influx apply -c default
```

Comment:
We are using config profiles here to export all the resources from the old instance and applying them to your new instance. The only things that will not be copied over are scraper configurations. You will need to manually reconfigure those.